test/async-request-transform.test.js

```diff
@@ -88,21 +86,20 @@ test('transformers should run serially', t => {
   x.addAsyncRequestTransform(req => {
     return new Promise((resolve, reject) => {
       setImmediate(_ => {
-        t.is(first, true)
-        t.is(second, false)
+        expect(first).toBeTruthy()
```

Comment:
I think we want stronger than truthy / falsy here. `t.is` expects actual true/false, so the equivalent would be `.toBe(true)`, yeah?